lib/quiet_safari.rb

```diff
@@ -14,7 +14,7 @@ class Engine < ::Rails::Engine
       Rails::Rack::Logger.class_eval do
         def call_with_quiet_safari(env)
           begin
-            if Rails.application.config.quiet_safari && env['PATH_INFO'] =~ APPL
+            if Rails.application.config.quiet_safari and env['PATH_INFO'] =~ APPL
```

Comment:
Oh - I must have changed that out of habit... I typically use "and" and "or" for clarity.